hibernate-core/src/main/java/org/hibernate/jpa/spi/NativeQueryArrayTransformer.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.jpa.spi;
+
+import org.hibernate.query.TupleTransformer;
+
+import java.util.List;
+
+/**
+ * A {@link TupleTransformer} for handling {@link Object[]} results from native queries.
+ */
```

Comment:
```suggestion
 *
 * @since 7.0
 */
```

Reply:
applied in https://github.com/hibernate/hibernate-orm/pull/10178